spring-batch-core/src/test/java/org/springframework/batch/core/step/tasklet/MethodInvokingTaskletAdapterTest.java

```diff
@@ -0,0 +1,239 @@
+package org.springframework.batch.core.step.tasklet;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.batch.core.ExitStatus;
+import org.springframework.batch.core.StepContribution;
+import org.springframework.batch.core.StepExecution;
+import org.springframework.batch.core.scope.context.ChunkContext;
+import org.springframework.batch.repeat.RepeatStatus;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.mock;
+
+public class MethodInvokingTaskletAdapterTest {
```

Comment:
Thanks for adding the explicit test for this class.  The tests we had for this class were integration tests through the `TaskletParserAdapterTests`.